### PR TITLE
STP-3181-1.2: Updated language in ConMan intro topic

### PR DESCRIPTION
**Pull request by nrockershousen**
## Summary and Scope

Removing the new in release 1.5 verbiage in the ConMan introduction topic for CSM 1.2+ releases.

## Issues and Related PRs

* Resolves [STP-3181](https://jira-pro.its.hpecorp.net:8443/browse/STP-3181)
* Merge with https://github.com/Cray-HPE/docs-csm/pull/1558

## Testing

N/A

### Tested on:

N/A

### Test description:

N/A

## Risks and Mitigations

None.

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Review by mharding-hpe — APPROVED** [review]

File: operations/conman/ConMan.md [hunk 1]
```diff
@@ -2,12 +2,13 @@
 
 ConMan is a tool used for connecting to remote consoles and collecting console logs. These node logs can then be used for various administrative purposes, such as troubleshooting node boot issues.
 
-ConMan runs on the system as a containerized service. It runs in a set of Docker containers within Kubernetes pods named `cray-console-operator` and `cray-console-node`. ConMan can be configured to encrypt usernames and passwords. Node console logs are stored locally within the `cray-console-operator pod` in the `/var/log/conman/` directory, as well as being collected by the System Monitoring Framework \(SMF\).
+ConMan runs on the system as a containerized service. It runs in a set of Docker containers within Kubernetes pods named `cray-console-operator` and `cray-console-node`.
+ConMan can be configured to encrypt usernames and passwords.
```
Comment (dlaine-hpe) [inline operations/conman/ConMan.md:6]: Not sure if this is true.  If it is I am not sure how to do it...

File: operations/conman/ConMan.md [hunk 2]
```diff
@@ -2,12 +2,13 @@
...
+Node console logs are stored locally within the `cray-console-operator pod` in the `/var/log/conman/` directory, as well as being collected by the System Monitoring Framework \(SMF\).
```
Comment (dlaine-hpe) [inline operations/conman/ConMan.md:7]: Change 'cray-console-operator pod' to 'cray-console-node-0 pod'.
Both are true (they point to the same mount), but in csm-1.3 the access will be removed from cray-console-operator so may as well start changing docs now.

File: operations/conman/ConMan.md [hunk 3]
```diff
@@ -2,12 +2,13 @@
...
 
-## New in Release 1.5
-
-In previous releases, the entire service was contained in the `cray-conman` pod, and both logs and interactive access were handled through that single pod. Starting with version 1.5, the logs can be accessed through the `cray-console-operator` pod, but interactive console access is handled through one of the `cray-console-node` pods. There are multiple `cray-console-node` pods, scaled to the size of the system.
+In CSM versions 1.0 and later, the ConMan logs are accessible through the `cray-console-operator` pod, but interactive console access is handled through one of the `cray-console-node` pods.
```
Comment (dlaine-hpe) [inline operations/conman/ConMan.md:9]: As above, may as well start changing the docs to point people to get the logs through 'cray-console-node'.  Maybe something like:
"In CSM versions 1.0 and later the ConMan logs and interactive consoles are accessible through one of the 'cray-console-node' pods."

**Review by dlaine-hpe — APPROVED** [review]
Just a few suggestions, but I will approve as they are optional if you want to make the changes or not.

File: operations/conman/ConMan.md [hunk 4]
```diff
@@ -2,12 +2,12 @@
 
 ConMan is a tool used for connecting to remote consoles and collecting console logs. These node logs can then be used for various administrative purposes, such as troubleshooting node boot issues.
 
-ConMan runs on the system as a containerized service. It runs in a set of Docker containers within Kubernetes pods named `cray-console-operator` and `cray-console-node`. ConMan can be configured to encrypt usernames and passwords. Node console logs are stored locally within the `cray-console-operator pod` in the `/var/log/conman/` directory, as well as being collected by the System Monitoring Framework \(SMF\).
+ConMan runs on the system as a containerized service. It runs in a set of Docker containers within Kubernetes pods named `cray-console-operator` and `cray-console-node`.
+Node console logs are stored locally within the `cray-console-node-0 pod` in the `/var/log/conman/` directory, as well as being collected by the System Monitoring Framework \(SMF\).
```
Comment (dlaine-hpe) [inline operations/conman/ConMan.md:6]: Not to be picky, but should the word 'pod' be inside the ``?

Comment (dlaine-hpe) [reply operations/conman/ConMan.md:6]: Come to think of it, may be better to say "within the `cray-console-node` pods in the `/var/log/conman/` directory.

**Review by dlaine-hpe — APPROVED** [review]

**Pull request merged**